parlai/mturk/tasks/acute_eval/README.md

```diff
@@ -1,255 +1,9 @@
-# ACUTE-Eval
+*** TODO: revise this ***
```

Comment:
is this still a TODO?

Reply:
Whoops! It wasn't a TODO, just forgot to remove that line - fixed